vllm/config.py

```diff
@@ -294,12 +294,20 @@ def compute_hash(self) -> str:
         factors.append(self.quantization)
         factors.append(self.revision)
         factors.append(self.code_revision)
+        factors.append(self.max_model_len)
+        factors.append(self.quantization)
+        factors.append(self.enforce_eager)
```

Comment:
Actually do we need to put force eager here?

Reply:
Will remove